pennylane/measurements/counts.py

```diff
@@ -320,6 +325,6 @@ def circuit(x):
         for result, outcome_dict in zip(results, outcome_dicts):
             states, _counts = result
             for state, count in zip(qml.math.unwrap(states), _counts):
-                outcome_dict[state] = count
+                outcome_dict[state] = int(count)
```

Comment:
i think this is a breaking change. was it necessary for something else? the tests reflect this break as well

Reply:
Another artifact. The values of the dict returned by `counts` is always wrapped in a pennylane numpy tensor, so I was casting it to try and get a nicer output, but it didn't make a difference. Regardless, nice catch! I'll remove it.